comp/networkpath/npcollector/npcollectorimpl/config.go

```diff
@@ -28,6 +29,7 @@ func newConfig(agentConfig config.Component) *collectorConfigs {
 		workers:                      agentConfig.GetInt("network_path.collector.workers"),
 		pathtestInputChanSize:        agentConfig.GetInt("network_path.collector.input_chan_size"),
 		pathtestProcessingChanSize:   agentConfig.GetInt("network_path.collector.processing_chan_size"),
+		pathtestMaxContexts:          agentConfig.GetInt("network_path.collector.pathtest_max_contexts"),
```

Comment:
Do we need to provide a default somewhere? Afaict this will default to 0, which is never the right configuration. 

For stuff like this I usually size at 150% of whatever p99 we're seeing if that number is available and easily accessible.

Reply:
Ah I missed that, never mind then, that feels like a reasonable approach